pkg/synthetictests/duplicated_events.go

```diff
@@ -199,13 +237,94 @@ func (d duplicateEventsEvaluator) testDuplicatedEvents(testName string, flakeOnl
 
 		if flake || flakeOnly {
 			flakes = append(flakes, appendToFirstLine(msg, " result=allow "))
+			fmt.Println("Found patho flake")
```

Comment:
patho to pathological if you plan to keep this logging, and include the message would be nice and clear.